### ISSUE TITLE
Redmi AC2100 5Ghz WiFi

### ISSUE THREAD

**Issue by CessXP** [hanwckf/rt-n56u#535]
I want to report some problems with the 5ghz connection in the RM AC2100, if I'm near of the router the connecting is great and works really good but if I move some meters the problem begins, I'm still connected to the WiFi but there are a lot of lost packages.

![Screenshot_20201231-012644~2](https://user-images.githubusercontent.com/31802781/103419992-f8e79b00-4b5a-11eb-9bcb-838786c9c4d4.png)

I tested every configuration available trying to solve this problem but is impossible, 2.4Ghz WiFi is really good.

I'm using USA channels and manually selected the channel 36 in 5Ghz.

**Comment by CessXP** [comment]
Tested again with a new build with latest commits, nothing changed, same problems :c

**Comment by jinglei207** [comment]
Does the problem happen also with other padavan firmware from different author?

**Comment by CessXP** [comment]
> Does the problem happen also with other padavan firmware from different author?

Yes, I tested zbancam ( russian ) , from hanwckf,ChongshengB and TurBoTs, and old compilations too from all of them.

A guy tell me maybe is driver issue.
